docs/resources/v2-conformance-statement.md

```diff
@@ -0,0 +1,818 @@
+| (0008, 1197) | FailureReason            | The reason code why this instance failed to store. |
+| (0074, 1048) | FailedAttributesSequence | The sequence of `ErrorComment` that includes the reason for each failed attribute. |
+
+Each dataset in the `ReferencedSOPSequence` will have the following elements:
+
+| Tag          | Name                     | Description |
+| :----------- | :----------------------- | :---------- |
+| (0008, 1150) | ReferencedSOPClassUID    | The SOP class unique identifier of the instance that was stored. |
+| (0008, 1155) | ReferencedSOPInstanceUID | The SOP instance unique identifier of the instance that was stored. |
+| (0008, 1190) | RetrieveURL              | The retrieve URL of this instance on the DICOM server. |
+
+An example response with `Accept` header `application/dicom+json`:
+
+```json
+{
+  "00081190":
+  {
+    "vr":"UR",
+    "Value":["http://localhost/studies/d09e8215-e1e1-4c7a-8496-b4f6641ed232"]
+  },
+  "00081198":
+  {
+    "vr":"SQ",
+    "Value":
+    [{
+      "00081150":
+      {
+        "vr":"UI","Value":["cd70f89a-05bc-4dab-b6b8-1f3d2fcafeec"]
+      },
+      "00081155":
+      {
+        "vr":"UI",
+        "Value":["22c35d16-11ce-43fa-8f86-90ceed6cf4e7"]
+      },
+      "00081197":
+      {
+        "vr":"US",
+        "Value":[43265]
+      }
+    }]
+  },
+  "00081199":
+  {
+    "vr":"SQ",
+    "Value":
+    [{
+      "00081150":
+      {
+        "vr":"UI",
+        "Value":["d246deb5-18c8-4336-a591-aeb6f8596664"]
+      },
+      "00081155":
+      {
+        "vr":"UI",
+        "Value":["4a858cbb-a71f-4c01-b9b5-85f88b031365"]
+      },
+      "00081190":
+      {
+        "vr":"UR",
+        "Value":["http://localhost/studies/d09e8215-e1e1-4c7a-8496-b4f6641ed232/series/8c4915f5-cc54-4e50-aa1f-9b06f6e58485/instances/4a858cbb-a71f-4c01-b9b5-85f88b031365"]
+      }
+    }]
+  }
+}
+```
+
+### Store Failure Reason Codes
+
+| Code  | Description |
+| :---- | :---------- |
+| 272   | The store transaction did not store the instance because of a general failure in processing the operation. |
+| 43264 | The DICOM instance failed the validation. |
+| 43265 | The provided instance StudyInstanceUID did not match the specified StudyInstanceUID in the store request. |
+| 45070 | A DICOM instance with the same StudyInstanceUID, SeriesInstanceUID and SopInstanceUID has already been stored. If you wish to update the contents, delete this instance first. |
+| 45071 | A DICOM instance is being created by another process, or the previous attempt to create has failed and the cleanup process has not had chance to clean up yet. Please delete the instance first before attempting to create again. |
+| 45063 | A DICOM instance Data Set does not match SOP Class. The Studies Store Transaction (Section 10.5) observed that the Data Set did not match the constraints of the SOP Class during storage of the instance. |
```

Comment:
Looks like this was moved from warnings to failure reasons, is that correct?

Reply:
No, I did not.

Reply:
Thanks for the feedback Esma, I am looking at this PR as incremental change targeting the change I made to V2 QIDO only. Rest needs to be tracked separately